jag-crdp-application/src/main/java/ca/bc/gov/open/crdp/controllers/TransmitController.java

```diff
@@ -56,10 +56,14 @@ public TransmitController(
         this.objectMapper = objectMapper;
     }
 
+    // Source from :    http://reeks.bcgov/
```

Comment:
source differs, might not be a good idea to add it here

Reply:
removed source